docs-ref-conceptual/upcoming-breaking-changes.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Upcoming Breaking Changes – Azure CLI | Microsoft Docs
+description: Learn about breaking changes coming to Azure CLI in the next breaking change release
+author: dbradish-microsoft
+ms.author: dbradish
+manager: barbkess
+ms.date: 08/22/2022
+ms.topic: article
+ms.service: azure-cli
+ms.tool: azure-cli
+ms.custom: devx-track-azurecli, seo-azure-cli
```

Comment:
are these markdown metadata that required or could be dropped?

Reply:
They are required by the docs platform. The change should be that her manager is now Michael Kluck and we should reflect that.